test/tests/markers/boolean_combo/tests

```diff
@@ -0,0 +1,37 @@
+[Tests]
+  [prioritize_refine_and]
+    type = CSVDiff
+    input = 'prioritize_refine_and.i'
+    csvdiff = 'prioritize_refine_and_out.csv'
+    requirement = 'The system shall be able to prioritize refinement while ANDing markers together.'
+    issues = '#1025'
```

Comment:
For files like this, where the `issues` and `design` are repeated many times, please put them higher up in the Tests spec. Like this, they will be carried to sub-blocks.

```
[Tests]
  issues = ...
  design = ...
[]
```